mentorship ios/Home/SupportingViews/DetailListCell.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  DetailListCell.swift
+//  Created on 17/06/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import SwiftUI
+
+struct DetailListCell: View {
+    var requestData: HomeModel.HomeResponseData.RequestStructure
+    var index: Int
+
+    var dateFormatter: DateFormatter {
```

Comment:
Strictly speaking dateformatter and processing the date should go int the ViewModel not the view. 

You could put it in the utilities for now?